### ISSUE TITLE
Homebridge-TPLink-Smarthome 6.4.0 Causes Crashing for HOOBS 3.3.5

### ISSUE THREAD

**Issue by mickelsn** [plasticrake/homebridge-tplink-smarthome#200]
It appears that v6.4.0 of this plugin is incompatible with the current release version of HOOBS despite being "HOOBS Certified". Based on the description provided below, I suspect that this version of the plugin requires a more current version of homebridge (1.3.3) than is included in the current HOOBS distribution. This version of the plugin causes the HOOBS bridge service to stop when the plugin initializes. Can the plugin be updated to check for a homebridge version lower than 1.3.1 and avoid use of 1.3.3-only features to try and avoid the crashing?

I am also reporting this issue to HOOBS to try and encourage them to (1) update ASAP to the latest version of homebridge, and (2) allow end-users to download and install prior versions of plugins to avoid these kinds of issues in future.

### Expected Behavior
BUG: Plugin should initialize without error and allow HOOBS/homebridge service to start.

### Current Behavior
BUG: Plugin attempts to initialize but fails; HOOBS/homebrdge service stops.

### Steps to Reproduce (for bugs)
1. Install plugin into an installation of HOOBS 3.3.5
2. View log in Dashboard to see the following:
```
5/22/2021, 11:13:25 PM [TplinkSmarthome] Initializing TplinkSmarthome platform...
5/22/2021, 11:13:25 PM [TplinkSmarthome] homebridge-tplink-smarthome v6.4.0, node v12.19.0, homebridge v1.3.3, api v2.7
```

### Versions

- **Node**:       `node --version` - v12.19.0
- **HOOBS**:  `hoobs --version` - 3.3.5
- **Homebridge**: `homebridge --version`. (Per HOOBS 3.3.5 release notes, it is built on Homebridge 1.3.1)
- **OS**:         Raspbian 10

### Configuration

```json
        {
            "platform": "TplinkSmarthome",
            "plugin_map": {
                "plugin_name": "homebridge-tplink-smarthome"
            }
        }
```

### Homebridge Log / Command Output

```
5/22/2021, 11:13:25 PM [TplinkSmarthome] Initializing TplinkSmarthome platform...
5/22/2021, 11:13:25 PM [TplinkSmarthome] homebridge-tplink-smarthome v6.4.0, node v12.19.0, homebridge v1.3.3, api v2.7
```

**Comment by plasticrake** [comment]
I have no affialition with HOOBS and I don't have anything to do with it being "certified". I recommending using regular homebridge and config-ui-x

**Comment by mickelsn** [comment]
Understand. Filed the report here in case you were interested, and also because it was required in the bug report I filed with HOOBS on this issue. In the meantime, I've switched over to native Homebridge and things work exactly as expected. Really, the one thing I wish Homebridge let me do that HOOBS added is restart the bridge server without rebooting my entire Raspberry Pi...

Thanks again for the effort on making this work!
